examples/comment-bot/arthas-api/api/post/topic/index.js

```diff
@@ -50,5 +60,75 @@ module.exports = asyncCache => async (req, res) => {
       res.end(JSON.stringify({
         success: true
       }));
+
+      // The bot should parse the question if its name is
+      // mentioned in the topic or supporting text
+
+      let query;
+
+      if (topic.toLowerCase().split(' ').includes(personaName.toLowerCase())) {
+        query = topic;
+      } else if (text.toLowerCase().split(' ').includes(personaName.toLowerCase())) {
+        query = text;
+      }
+
+      if (query) {
```

Comment:
This would be the `onCommand` / `onMessage` handling code in a chat bot like those on Slack or Discord. 

In this example, posting the forum comment uses the same `/v1/comment` endpoint like any other forum user would, providing the `answer.text` as the comment `text`.